addons/mrp/report/mrp_production_templates.xml

```diff
@@ -81,65 +71,33 @@
                     </h3>
 
                     <table class="table table-sm" t-if="o.move_raw_ids">
-                        <t t-set="has_location" t-value="any(o.move_raw_ids.filtered(lambda x: x.move_line_ids.filtered(lambda x: x.location_id.id)))" groups="stock.group_stock_multi_locations"/>
-                        <t t-set="has_serial_number" t-value="any(o.move_raw_ids.filtered(lambda x: x.move_line_ids.filtered(lambda x: x.lot_id or x.lot_name)))" groups="stock.group_production_lot"/>
                         <t t-set="has_product_barcode" t-value="any(o.move_raw_ids.filtered(lambda x: x.product_id.barcode))"/>
-                        <t t-set="has_product_available" t-value="any(o.move_raw_ids.filtered(lambda x: x.move_line_ids.filtered(lambda x: x.product_qty)))"/>
-                        <t t-if="o.state=='done'">
-                            <t t-set="has_product_available" t-value="any(o.move_raw_ids.filtered(lambda x: x.move_line_ids.filtered(lambda x: x.qty_done)))"/>
-                        </t>
                         <thead>
-                            <tr t-if="has_product_available">
+                            <tr>
                                 <th>Product</th>
-                                <th>Quantity</th>
-                                <th><span t-if="has_location">From</span></th>
-                                <th t-if="has_serial_number" width="15%">Lot/Serial barcode</th>
+                                <th t-attf-class="#{'text-right' if not has_product_barcode else ''}">Quantity</th>
```

Comment:
I would rather to use the odoo syntax {{ }} instead of jinja syntax #{}. Could you modify it for both t-attf-class